src/conversions/chrono.rs

```diff
@@ -0,0 +1,266 @@
+#![cfg(feature = "chrono")]
```

Comment:
If there are any `Error` types in `chrono`, it may be nice to also implement `From<Error> for PyErr` for them.